templates/webapi/main/group_builds.html.ep

```diff
@@ -45,7 +45,7 @@
                             </span>
                         </div>
                         <div class="px-2 align-self-end flex-grow-1">
-                            %= include 'main/build_progressbar', max_jobs => $jobs, result => $child_res
+                            %= include 'main/build_progressbar', max_jobs => $jobs, result => $child_res, class => ""
```

Comment:
I'd use single quotes so at least the quoting within that line is consistent.

Reply:
yes indeed